<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Olivier Trichet &lt;nive@nivalis.org&gt;</author>
 <class>GroupSelectionDialog</class>
 <widget class="KDialog" name="GroupSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>714</height>
   </rect>
  </property>
  <widget class="QWidget" name="page" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>559</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="_2" stretch="0,1">
    <item>
     <widget class="QWidget" name="mFilterContainer" native="true">
      <layout class="QHBoxLayout" name="_3">
       <item>
        <widget class="QLabel" name="mFilterLabel">
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="buddy">
          <cstring>mFilterLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="mFilterLineEdit">
         <property name="trapEnterKeyEvent" stdset="0">
          <bool>true</bool>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>mFilterLabel</zorder>
      <zorder>mFilterLineEdit</zorder>
      <zorder>mViewSplitter</zorder>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="mViewSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="Akonadi::EntityTreeView" name="mGroupsView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QGridLayout" name="mChangeViewLayout">
        <item row="0" column="0" rowspan="2">
         <widget class="QWidget" name="mButtonContainer" native="true">
          <layout class="QVBoxLayout" name="_4" stretch="4,0,1,0,10">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="mButtonSpacerTop">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>112</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="KPushButton" name="mRevertFromGroupViewButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="mButtonSpacerMiddle">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>26</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="KPushButton" name="mRevertFromChangeViewButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="mButtonSpacerBottom">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>285</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="mChangeViewLabel">
          <property name="text">
           <string>Current changes:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTreeView" name="mChangeView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDialog</class>
   <extends>QDialog</extends>
   <header>kdialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Akonadi::EntityTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">Akonadi/EntityTreeView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
